shared/charts/gpii-flushtokens/templates/cronjob.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  namespace: {{ .Release.Namespace | quote }}
+  name: {{ template "flushtokens.name" . }}
+spec:
+  schedule: "*/15 * * * *"
+  concurrencyPolicy: Allow
```

Comment:
Yes, it does seem silly to have allow concurrency.  I have a very vague recollection that, very early on, something went wrong when I set it to `Forbid`, but I can't find any notes about it, and it may be a false memory.  Also, the code has changed a since then, so I'll change it to `Forbid` and see what happens.